docs/Custom_Rules.mdx

```diff
@@ -14,7 +14,7 @@ The main three changes to custom rules writing are:
 
 - [Utils Package](#utils-package): we recommend using `@typescript-eslint/utils` to create custom rules
 - [AST Extensions](#ast-extensions): targeting TypeScript-specific syntax in your rule selectors
-- [Typed Rules](#typed-rules): using the TypeScript type checker to inform rule logic
+- [Typed Rules](#type-checking): using the TypeScript type checker to inform rule logic
```

Comment:
I think the right fix would be the other way around - that the section should be named _Typed Rules_ not _Type Checking_. Could you make that fix instead please?

Reply:
Thank you so much for your kind response... I fixed it as you mentioned.